ui/raidboss/data/02-arr/raid/t9.js

```diff
@@ -162,10 +162,10 @@
     },
     {
       id: 'T9 Garotte Twist Lose',
-      regex: / 1E:\y{ObjectId}:(\y{Name}) loses the effect of Garrote Twist/,
-      regexDe: / 1E:\y{ObjectId}:(\y{Name}) loses the effect of Leicht Fixierbar/,
-      regexFr: / 1E:\y{ObjectId}:(\y{Name}) loses the effect of Sangle Accélérée/,
-      regexJa: / 1E:\y{ObjectId}:(\y{Name}) loses the effect of 拘束加速/,
+      regex: / 1E:\y{ObjectId}:\y{Name} loses the effect of Garrote Twist/,
```

Comment:
I think this is meant to have a conditional that this is you, because `T9 Garotte Twist Gain` only applies to you.

Reply:
I updated this to be what I thought logically made sense; ie the inverse of gain. Please check when you can 😄